content/200-concepts/100-components/02-prisma-client/240-metrics.mdx

```diff
@@ -38,7 +38,7 @@ Prisma Client provides the following metrics:
   - `prisma_client_queries_wait`: The number of Prisma Client queries currently waiting for a connection because all connections are in use.
   - `prisma_pool_connections_busy`: The number of currently busy pool connections. These pool connections are currently executing a datasource query.
   - `prisma_pool_connections_idle`: The number of pool connections that are not currently being used. These pool connections are waiting for the next datasource query to run.
-  - `prisma_pool_connections_open`: The total number of [pool connections](/concepts/components/prisma-client/working-with-prismaclient/connection-pool#default-connection-pool-size) opened.
+  - `prisma_pool_connections_open`: The number of [pool connections](/concepts/components/prisma-client/working-with-prismaclient/connection-pool#default-connection-pool-size) opened.
```

Comment:
This is still not the number of "opened", but currently "open".

Doesn't this description come from the API when using it?

Reply:
Got it.

Side note: There are duplicate properties available, which I'm not sure if it's intentional or not.

```
    {
      "key": "prisma_pool_connections_open",
      "labels": {
        
      },
      "value": 1,
      "description": "Number of currently open Pool Connections"
    },
    {
      "key": "prisma_pool_connections_open",
      "labels": {
        
      },
      "value": 0,
      "description": "Number of currently open Pool Connections (able to execute a datasource query)"
    }
```

Reply:
Added it to my TODO to create an issue for it once I get a reproduction after the release.